changelog.md

```diff
@@ -65,6 +65,27 @@
 
 - `echo` and `debugEcho` will now raise `IOError` if writing to stdout fails.  Previous behavior
   silently ignored errors.  See #16366.  Use `-d:nimLegacyEchoNoRaise` for previous behavior.
+- Added `lists.toSinglyLinkedList` and `lists.toDoublyLinkedList` for conversion from
```

Comment:
changelog needs to be fixed (reason for duplication is `/changelog.md merge=union`, but it's there for a good reason)